rest/models/src/main/java/com/connexta/store/rest/models/QuarantineRequest.java

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
:question: I see a QuarantineRequest, but no PromoteRequest. If that scope part of a separate tickets? Does it mean you've decided to have two endpoints-- one for quarantine and one for promote? Or will one endpoint service both kinds of requests?

Reply:
addMetadata functionally is promote.